hwy/ops/x86_128-inl.h

```diff
@@ -4497,9 +4505,15 @@ HWY_API VFromD<D> Reverse2(D d, VFromD<D> v) {
 
 template <class D, typename T = TFromD<D>, HWY_IF_T_SIZE(T, 2)>
 HWY_API VFromD<D> Reverse2(D d, VFromD<D> v) {
-#if HWY_TARGET <= HWY_AVX3 || HWY_TARGET == HWY_SSE2
+#if HWY_TARGET <= HWY_AVX3
   const Repartition<uint32_t, decltype(d)> du32;
   return BitCast(d, RotateRight<16>(BitCast(du32, v)));
+#elif HWY_TARGET == HWY_SSE2
+  constexpr size_t kN = MaxLanes(d);
+  __m128i shuf_result = _mm_shufflelo_epi16(v.raw, _MM_SHUFFLE(2, 3, 0, 1));
+  if(kN > 4)
```

Comment:
A request for future code: we use clang-format with the Google style. Please generally write braces for if-conditions unless it's something trivial such as a return or break that fits in one line.

So:
```
if (kN > 4) {
  shuf_result = ..;
}
```
No worries about this one, will update it the next time I edit this file.

Reply:
hm, looks like that mainly validates/checks the code - or does it also send pull requests? If not, we are mostly covered already because our IDE does display lint warnings, and also auto-formats when saving that file.